acid-view/src/CmdParse.hs

```diff
@@ -0,0 +1,64 @@
+module CmdParse where
+
+import Options.Applicative
+import System.Directory (getHomeDirectory)
+
+data TypeOfOutput = Wallets | Accounts | Addresses | AllData
+  deriving Show
+
+data CmdOptions = CmdOptions
+    {  path       :: FilePath
+    ,  position   :: Int
+    ,  output     :: TypeOfOutput
+    ,  fileToSave :: Maybe FilePath
+    }
+
+
+cmdArgToTypeOfOutput :: String -> Either String TypeOfOutput
+cmdArgToTypeOfOutput output = case output of
+    "wal"  -> Right Wallets
+    "acc"  -> Right Accounts
+    "addr" -> Right Addresses
+    "all"  -> Right AllData
+    _      -> Left "Can't parse type of output"
+
+opts :: FilePath -> ParserInfo CmdOptions
+opts path = info ((fileOptions path) <**> helper)
+      ( fullDesc
+     <> progDesc "Show acid state db contents"
+     <> header "Show acid state db contents" )
```

Comment:
Let's go with `Show Acid DB contents` or `Show acid-state DB contents`, preferably the former.